### PR TITLE
[doc] Update getting started docs for latest release

### PR DESCRIPTION
**Pull request by GregAC**
Updated the docs linked from getting started to match what's needed for the v0.3 release, note the downgrade on the baud rate (things built with sonata-software are still running on 115200).

I'd appreciate a rapid review on this, the prior v0.2 release was effectively broken because the latest main of sonata-software wouldn't build binaries compatible with it (due to the switch to writing .elf directly to flash). We've got a new (working) v0.3 release done but want to ensure people reading the getting started docs are guided appropriately so they can use it.

There's more doc updates/reviews to do to get everything consistent for v1.0 this is just the critical stuff so the getting started material is consistent with the latest working release.

### CODE REVIEW

**Comment by GregAC** [comment]
Oh and once this is merged I think I'll bump the v0.3 tag so it includes these docs.

File: doc/guide/fpga-update.md
Comment (marnovandermaas) [inline doc/guide/fpga-update.md:34]:
```suggestion
2. Plug in your Sonata board using the main USB. You should see a `SONATA` drive (see troubleshooting section if unsure).
```

File: doc/guide/fpga-update.md
Comment (marnovandermaas) [inline doc/guide/fpga-update.md:36]:
```suggestion
4. Make sure that you have the bitstream switch (SW3) set to 1.
5. The board should automatically restart once the image is copied over. You should see the `FPGA Config` LED come on:
```

**Review by marnovandermaas — APPROVED** [review]
A few suggestions from my end but otherwise looks good. Will approve so that once you've made the changes you can merge directly.

**Comment by GregAC** [comment]
Thanks for the review @marnovandermaas

**Pull request merged**
